### ISSUE TITLE
[rollbar] Unhandled exception: Unhandled Promise rejection: editCache Error: command 'vscode.previewHtml' not found {}

### ISSUE THREAD

**Issue by lygstate** [vector-of-bool/vscode-cmake-tools#676]
https://github.com/Microsoft/vscode/issues/62630

### Expected:

1. Push button
2. Thing happens

### Apparent Behavior:

1. Push button
2. Nothing happens

### CMake Tools Log

```
<!-- Paste the log contents HERE -->
```

### Developer Tools Log

```
<!-- Paste the log contents HERE -->
```

### Platform and Versions

- **Operating System**: 
- **CMake Version**: 
- **VSCode Version**: 
- **CMake Tools Extension Version**: 
- **Compiler/Toolchain**:

**Comment by KoeMai** [comment]
Fix is already in development branch by PR #647.

**Comment by christianparpart** [comment]
I'm on VSCode (on Win10) with WSL. with all up-to-day (i.e. no software tells me to update) and yet I got this message, despite it being fixed and this ticked closed already. What I did to trigger this error message, was Ctrl+Shift+P -> "Edit cmake cache".

**Comment by bobbrow** [comment]
This was fixed (the API was removed from recent versions of VS Code), but an update with the fix has not yet been released.  It will be fixed in 1.2.0.

If you need a fix now, you can patch the JavaScript in your extensions directory on your local machine. 
 The path depends on OS and whether you are using the Remote - WSL extension.  It's usually something like `~/.vscode/extensions` or `~/.vscode-server/extensions` on WSL:

**${pathToVscodeExtensions}\vector-of-bool.cmake-tools-1.1.3\out\src\cmake-tools.js**
Replace line 718:
```
        await vscode.commands.executeCommand('vscode.previewHtml', 'cmake-cache://' + drv.cachePath, vscode.ViewColumn.Three, 'CMake Cache');
```
with the following:
```
        vscode.workspace.openTextDocument(vscode.Uri.file(drv.cachePath)).then(doc => {
            vscode.window.showTextDocument(doc);
        });
```
